docs/_pages/cookie-policy/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: subpage
+permalink: /cookie-policy/
```

Comment:
I just realized that having a copy/pasted version of our cookie policy on gruntwork.io, terragrunt.gruntwork.io, and terratest.gruntwork.io is going to make maintenance painful. Could you replace this with a link directly to https://gruntwork.io/cookie-policy/? Same for the Terragrunt website?